lib/plugins/install/install.js

```diff
@@ -22,6 +24,11 @@ class Install {
             required: true,
             shortcut: 'u',
           },
+          name: {
+            usage: 'Name for the service',
+            required: false,
```

Comment:
This could be left out as it means that same as `require: false` (AFAIK we don't follow this pattern in other plugins).